apps/v3_api/lib/trips.ex

```diff
@@ -14,4 +14,18 @@ defmodule V3Api.Trips do
     opts = put_in(opts[:route], route_id)
     get_json("/trips/", opts)
   end
+
+  def short_name(stop_name) do
```

Comment:
suggestion(non-blocking): This is mostly a style thing, nothing wrong with what you have here so take it or leave it, but just to mention another "Elixir-y" way to accomplish this might be to define a separate function for each potential value of `stop_name` and then not need the case statement. I.e.:

		def short_name("Needham Heights") do
			"NEED"
		end

		…

		def short_name(stop_name) do
			String.upcase(String.slice(stop_name, 0..2))
		end

Completely unrelated code, but just because I was looking at it earlier today, there's a similar example in [`mailer.ex`](https://github.com/mbta/dotcom/blob/master/apps/feedback/lib/mailer.ex#L77).